### PR TITLE
code for the api documentation

### PR DESCRIPTION
**Pull request by solonkonora**

### CODE REVIEW

File: public/api-docs/index.yaml [hunk 1]
```diff
@@ -7,13 +7,188 @@ info:
...
+        '500':
+          description: "Server error"
+
+  /match/publish:
+    post:
+      summary: "Publish a match between a form and an audio file"
+      description: "Creates a new match between a form and an audio file, along with the user who published it."
+      requestBody:
+        required: true
+        content:
```
Comment (NgoranA) [inline public/api-docs/index.yaml:77]: We need to tell frontend guys what the body should look like.

File: public/api-docs/index.yaml [hunk 2]
```diff
@@ -7,13 +7,188 @@ info:
...
 servers:
-  - url: http://localhost:8080
+  - url: http://localhost:8080/api/v1
     description: local server
 
-# paths:
-#   /:
-#     # define route paths
+paths:
+  /Languages:
```
Comment (NgoranA) [inline public/api-docs/index.yaml]: "L" should be l .

File: public/api-docs/index.yaml [hunk 3]
```diff
@@ -7,13 +7,188 @@ info:
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PublishMatchResponse'
+        '400':
+          description: "Invalid input"
+        '500':
+          description: "Server error"
+
+  /login:
```
Comment (NgoranA) [inline public/api-docs/index.yaml]: auth/login

File: public/api-docs/index.yaml [hunk 4]
```diff
@@ -7,13 +7,188 @@ info:
...
+    post:
+      summary: "User login"
+      description: "Logs in a user and returns an authentication token."
+      requestBody:
+        required: true
+        content:
```
Comment (NgoranA) [inline public/api-docs/index.yaml:99]: same here as well, the frontend guys need to be told what to send to the backend.

**Review by NgoranA — CHANGES_REQUESTED** [review]
Make some changes as requested from me.

File: public/api-docs/index.yaml (hunk 1 above)
Comment (solonkonora) [reply public/api-docs/index.yaml:77]: working on it sir

File: public/api-docs/index.yaml (hunk 2 above)
Comment (solonkonora) [reply public/api-docs/index.yaml]: noted

File: public/api-docs/index.yaml (hunk 3 above)
Comment (solonkonora) [reply public/api-docs/index.yaml]: noted

File: public/api-docs/index.yaml (hunk 4 above)
Comment (solonkonora) [reply public/api-docs/index.yaml:99]: worked on.

**Comment by eugeneegbe** [comment]
@VictorNico @RashJrEdmund

**Comment by VictorNico** [comment]
Hi

**Comment by VictorNico** [comment]
The PR has to be closed due to unchanged. Please, apply the comment and create and new one. for that, new issues and PR will be required.

**Pull request closed without merging**